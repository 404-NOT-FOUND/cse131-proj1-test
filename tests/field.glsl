a.c; a.c; a. c; a.	c; a.
c;
a.c;
a.c;
a./* comment */c;
a.+c;
a.=c;
a.c d;
a.b.c.d;
