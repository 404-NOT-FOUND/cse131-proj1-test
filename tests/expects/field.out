a            line 1 cols 1-1 is T_Identifier 
.            line 1 cols 2-2 is T_Dot 
c            line 1 cols 3-3 is T_FieldSelection 
;            line 1 cols 4-4 is T_Semicolon 
a            line 1 cols 6-6 is T_Identifier 
.            line 1 cols 7-7 is T_Dot 
c            line 1 cols 8-8 is T_FieldSelection 
;            line 1 cols 9-9 is T_Semicolon 
a            line 1 cols 11-11 is T_Identifier 
.            line 1 cols 12-12 is T_Dot 
c            line 1 cols 14-14 is T_FieldSelection 
;            line 1 cols 15-15 is T_Semicolon 
a            line 1 cols 17-17 is T_Identifier 
.            line 1 cols 18-18 is T_Dot 
c            line 1 cols 25-25 is T_FieldSelection 
;            line 1 cols 26-26 is T_Semicolon 
a            line 1 cols 28-28 is T_Identifier 
.            line 1 cols 29-29 is T_Dot 
c            line 2 cols 1-1 is T_FieldSelection 
;            line 2 cols 2-2 is T_Semicolon 
a            line 3 cols 1-1 is T_Identifier 
.            line 3 cols 2-2 is T_Dot 
c            line 3 cols 3-3 is T_FieldSelection 
;            line 3 cols 4-4 is T_Semicolon 
a            line 4 cols 1-1 is T_Identifier 
.            line 4 cols 2-2 is T_Dot 
c            line 4 cols 3-3 is T_FieldSelection 
;            line 4 cols 4-4 is T_Semicolon 
a            line 5 cols 1-1 is T_Identifier 
.            line 5 cols 2-2 is T_Dot 
c            line 5 cols 16-16 is T_FieldSelection 
;            line 5 cols 17-17 is T_Semicolon 
a            line 6 cols 1-1 is T_Identifier 
.            line 6 cols 2-2 is T_Dot 
+            line 6 cols 3-3 is T_Plus 
c            line 6 cols 4-4 is T_Identifier 
;            line 6 cols 5-5 is T_Semicolon 
a            line 7 cols 1-1 is T_Identifier 
.            line 7 cols 2-2 is T_Dot 
=            line 7 cols 3-3 is T_Equal 
c            line 7 cols 4-4 is T_Identifier 
;            line 7 cols 5-5 is T_Semicolon 
a            line 8 cols 1-1 is T_Identifier 
.            line 8 cols 2-2 is T_Dot 
c            line 8 cols 3-3 is T_FieldSelection 
d            line 8 cols 5-5 is T_Identifier 
;            line 8 cols 6-6 is T_Semicolon 
a            line 9 cols 1-1 is T_Identifier 
.            line 9 cols 2-2 is T_Dot 
b            line 9 cols 3-3 is T_FieldSelection 
.            line 9 cols 4-4 is T_Dot 
c            line 9 cols 5-5 is T_FieldSelection 
.            line 9 cols 6-6 is T_Dot 
d            line 9 cols 7-7 is T_FieldSelection 
;            line 9 cols 8-8 is T_Semicolon 
